src/shared/assets.ts

```diff
@@ -345,6 +345,49 @@ export class AssetGenerator {
 
         return result;
     }
+
+    public getAssetsPathAndProgram(): [string, string] {
+        let assetsPath = ``;
+        this.executableProjects.forEach((project) => {
+            if (project.isWebAssemblyProject) {
+                assetsPath += path.join(path.dirname(project.outputPath), path.sep);
+                assetsPath += ';';
+            }
+        });
+        assetsPath = assetsPath.slice(0, -1);
+        return [assetsPath, this.executableProjects[0].outputPath];
+    }
+
+    public isDotNet9OrNewer(): boolean {
```

Comment:
>isDotNet9OrNewer [](http://example.com/codeflow?start=11&length=16)

Shouldn't we combine this together with `getAssetsPathAndProgram` so we only return assets for wasm projects that target .NET 9?